README.md

```diff
@@ -123,7 +101,30 @@ A related option is VS Code Dev Containers, which will open the project in your
 
     Note that this command will initialize a git repository, so you do not need to clone this repository.
 
-### Deploying
+### Quickstart
+
+#### Running locally
```

Comment:
I'm not sure why this was moved up, given it's not relevant til post-deployment?

Reply:
Please check the new order.